http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/AppServiceInterp.scala

```diff
@@ -33,6 +33,9 @@ import org.typelevel.log4cats.StructuredLogger
 
 import java.time.Instant
 import java.util.UUID
+
+import org.broadinstitute.dsde.workbench.leonardo.AppRestore.{CromwellRestore, GalaxyRestore}
```

Comment:
Shouldn't this import be up with the other org.broadinstitute imports?

Reply:
It seems IntelliJ auto import put it there, but I can move it up with other org.broadinstitute imports